mne/decoding/time_gen.py

```diff
@@ -61,7 +61,7 @@ def __repr__(self):
 
 
 class _GeneralizationAcrossTime(object):
-    """ see GeneralizationAcrossTime
+    """see GeneralizationAcrossTime
```

Comment:
never mind that. Ignore. Just make sure docstring starts with capital letter and ends with period.